standard.md

```diff
@@ -0,0 +1,136 @@
+
+
+### Required options
+
+- **alias**: `str`. Unique name for the dataset. Each (`alias`, `variable`) pair *must* be unique.
+- **filename**: `str`. Absolute path to the data file.
+- **variable**: `str`. Variable name to be used in the diagnostic. It is not necessary to follow CF conventions naming for variables.
+
+### Reserved options
+
+#### Related to the dataset
+
+- **project**: `str`. Name of the project to which a dataset belongs. It can be a real project, (e.g. `CMIP5`, `CMIP6`) or a tool-defined grouping of datasets
+(e.g. `Observations` or `Reanalysis`).
+- **activity**: `str`. CMIP activity, or analog, of a dataset.
+- **institute**: `str`. Name of the institute producing a dataset.
+- **dataset**: `str`. Name of the dataset. For example: the CMIP model identifier, the name of an observational dataset, etc.
+- **experiment**: `str`. Label to identify the experiment a dataset belongs to. (e.g `historical`, `piControl`).
+- **ensemble**: `str`. Ensemble identifier of a dataset.
+
+#### Related to the variable
+
+- **table**: `str`. Variable's original MIP table.
+- **original_frequency**: `str`. Variable's original frequency.
+- **frequency**: `str`. Variable's frequency when entering the diagnostic.
+- **modeling_realm**: `str`. Realms a variable belongs to.
+- **grid**: `str`. Variable's original grid.
+- **units**: `str`. Variable's units.
+- **short_name**: `str`. Variable's name used in the file as given by CF conventions.
+- **standard_name**: `str`. Standard name of the variable as given by CF conventions.
+- **long_name**: `str`. Long name of the variable as given by CF conventions.
+
+#### Others
+
+- **start**: `str`. Starting date of the data, given in ISO 8601 format for dates (YYYYMMDD).
+- **end**: `str`, Ending date of the data, in ISO 8601 format for dates (YYYYMMDD).
+- **start_year**: `int`. Starting year of the data.
+- **end_year**: `int`. End year of the data.
+- **reference_dataset**: `str`. Alias of the dataset to be used as the reference for comparisons.
+
+## The script formal description file
+
+It is *recommended* that each script comes with a description file in YAML syntax which includes information regarding: documentation, configuration of script, and optional settings.
+
+The following labels should be provided in order to complete the documentation section:
+
+- **title**: `str`. Label for the title of the diagnostic.
+- **description** `str`. A short description of the diagnostic.
+- **references** `list(str)`. List of references to be cited if using the diagnostic.
+- **authors** `list(str)`. List of authors that developed the diagnostic.
+- **read_more**: `url`. An URL to the online documentation.
+
+The labels listed below should be provided in order to complete the configuration of the script:
+
+- **script_name**: `str`. Label for the name of the script, which uniquely identifies it, and is delivered by an authoritative entity (such as ESGF, ENES, or WCRP)
```

Comment:
```suggestion
- **script_name**: `str`. Label for the name of the script, which uniquely identifies it
```
I don't think an entity exists which will issue script names, does it?

Reply:
I agree :  we need one. But we could make explicit which is the domain of validity of such names; is it only a tool-specific and tool-managed register ?

Reply:
> is it only a tool-specific and tool-managed register ?

Reply:
Fine !

Reply:
Added. And maybe the need of an official registry of script given by authoritative entities names is something that could be mentioned in the conclusions of the deliverable.